runtime/stdlib/contracts/test.cdc

```diff
@@ -346,4 +447,27 @@ pub contract Test {
         })
     }
 
+    /// Asserts that the result of an executed operation, such as
+    /// scripts & transactions, has errored and the error message
+    /// contains the given sub-string.
+    ///
+    pub fun assertError(_ result: {Result}, substr: String) {
+        pre {
+            result.status == ResultStatus.failed: "no error was found"
+        }
+
+        var found: Bool = false
```

Comment:
```suggestion
        var found = false
```

Reply:
Fixed in https://github.com/onflow/cadence/pull/2696/commits/600ed65a245e0d67a65f677c60c707fa047dfff1